app/webpacker/css/darkswarm/taxons.scss

```diff
@@ -13,66 +13,8 @@
 
   padding: 0.25rem 0.5rem 0.35rem 0.35rem;
 
-  render-svg {
-    svg {
-      width: 16px;
-      height: 16px;
-
-      path {
-        fill: #555;
-      }
-    }
-  }
-
   &, & * {
     display: inline-block;
     color: #555;
   }
 }
-
-.inactive {
-  .fat-taxons {
-    render-svg {}
-
-    svg {
-      path {
-        fill: $disabled-dark;
-      }
```

Comment:
Although this rule isn't scoped to render-svg, it probably was intended to be. It looks to me like there's never an `svg` in this context:
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/views/shops/_fat.html.haml#L13-L14
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/views/producers/_fat.html.haml#L25-L26